src/DataTable/useViewMore.ts

```diff
@@ -5,39 +5,40 @@ import * as React from "react"
  * triggered by a click event.
  */
 const useViewMore = () => {
+  let closeTimeout = 0
```

Comment:
First I thought it's a timeout duration, but then figured from the code below that it is actually a close timeout Id. Shall we rename it to avoid confusion?

```suggestion
  let closeTimeoutId = 0
```